chamge/amqp-edge-backend.c

```diff
@@ -11,9 +11,20 @@
 #include <gio/gio.h>
 #include <amqp.h>
 #include <amqp_tcp_socket.h>
+#include <json-glib/json-glib.h>
 
 #define AMQP_EDGE_BACKEND_SCHEMA_ID "org.hwangsaeul.Chamge1.Edge.AMQP"
 #define DEFAULT_CONTENT_TYPE "application/json"
+#define DEFAULT_CONTENT_TYPE_LEN 16
```

Comment:
If this length is required for string comparison,  `g_strcmp0()` is safer.
This way would be programmer error if content-type is changed.